preview-src/admonitions.adoc

```diff
@@ -1,4 +1,5 @@
 = Admonitions
+:page-editable: true
```

Comment:
Why did you change all pages to add the edit buttons? This is a lot of change and doesn't bring nothing
I suggest we revert it and we create 2 pages in the test sections to have one page with edit buttons and another one without buttons. We can keep the edit buttons on the index page.
What do you think about this proposal?

Reply:
I did that on pupose, to mimic the actual site.
This allowed me to find a bug with Message Blocks (Out of support, ...). So IMHO it's better to keep this.
We could have a single page without the icons for testing purpose though.

Reply:
You can add the edit buttons to the tests page that include the "custom message" block or "next versions" block instead. These 2 messages are intended to be used in page that always have the buttons.

Reply:
After discussions with Jerome, we decided to
- update the test pages involving block that are always marked as editable in production
- add a dedicated page for testing the display when the page is not editable
- make all other pages editable except the one for "out of support" which is never set as editable in production

Reply:
done